android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt

```diff
@@ -281,6 +297,25 @@ class ModelViewer(val engine: Engine) : android.view.View.OnTouchListener {
         return true
     }
 
+    private suspend fun fetchResources(callback: (String) -> Buffer) = withContext(Dispatchers.IO) {
```

Comment:
I don't think you need the `withContext(Dispatchers.IO)` here since you already launched the coroutine in the IO dispatcher.

Reply:
Nice!  Done.